src/main/java/org/mule/extension/aggregator/api/stereotype/AggregatorStereotype.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.aggregator.api.stereotype;
+
+import static org.mule.runtime.extension.api.stereotype.MuleStereotypes.PROCESSOR_DEFINITION;
+
+import org.mule.runtime.extension.api.stereotype.StereotypeDefinition;
+
+import java.util.Optional;
+
+/**
+ * {@link StereotypeDefinition} for a generic {@code aggregate} operation.
```

Comment:
Fix JavaDoc. aggregate is not the operation. @link to org.mule.extension.aggregator.internal.operations.AbstractAggregatorOperations